docs/tutorial/quickstart.md

```diff
@@ -12,16 +12,15 @@ Create a new Django project named `tutorial`, then start a new app called `quick
 
     # Create a virtualenv to isolate our package dependencies locally
     virtualenv env
-    source env/bin/activate  # On Windows use `env\Scripts\activate`
+    . env/bin/activate  # On Windows use `env\Scripts\activate`
```

Comment:
Ok, you guys are right - I remember my first confusion when I saw these dots in shell script.